var/spack/repos/builtin/packages/py-brian2/package.py

```diff
@@ -33,6 +36,7 @@ class PyBrian2(PythonPackage):
     depends_on('py-sphinx@1.5:', type=('build', 'run'), when='+docs')
     depends_on('py-sphinx@1.8:', type=('build', 'run'), when='@2.4:+docs')
     depends_on('py-ipython@5:', type=('build', 'run'), when='@2.4:+docs')
+    depends_on('py-sphinx-tabs', type=('build', 'run'), when='@2.5:+docs')
```

Comment:
I don't think this package exists in Spack, you'll need to write a package for this

Reply:
Sorry, should have added a "Depends on PR 27698"